frontend-src/index.html

```diff
@@ -2,10 +2,13 @@
 <html lang="en">
   <head>
     <meta charset="utf-8"/>
-    <title>Datasette Nteract Data Explorer</title>
+    <title>Datasette Nteract Data Explorer Demo</title>
   </head>
       <script type='module' src="main.tsx"></script>
   <body>
     <div id="root"></div>
+    <p class="export-links">This data as <a href="https://datasette-nteract-data-explorer.vercel.app/happy_planet_index.json?sql=select%20*%20from%20hpi_cleaned">json</a></p>
```

Comment:
Before this, there was no mount point. The stub was made during the earliest stage of development where the mount point HTML hadn't been chosen yet.